dcat/rdf/schema.ttl

```diff
@@ -15,6 +15,42 @@
 @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
+<http://purl.org/dc/elements/1.1/title>
+  owl:equivalentProperty schema:name ;
+.
+dcterms:description
+  owl:equivalentProperty schema:description ;
+.
+dcterms:format
```

Comment:
It seems to me that dcterms:format should be mapped to schema:fileFormat instead of schema:encodingFormat (that has domain schema:MediaObject)

Reply:
I took that straight from https://www.w3.org/2015/spatial/wiki/ISO_19115_-_DCAT_-_Schema.org_mapping#Distribution 
At this stage I didn't challenge the prior work, I merely encoded it. 
However, I did have to apply some judgement about whether to use rdfs:subPropertyOf or owl:equivalentProperty